samples/app-anonymous-users/nodejs/ClientApp/.env

```diff
@@ -0,0 +1,3 @@
+PORT = 3978
+REACT_APP_MICROSOFT_APP_ID="{{Microsoft-App-id}}"
```

Comment:
Do we need only these values?

Reply:
Yes, only the client's three values are required.
Server-side, I have added multiple values.